src/pages/ProductDetails/components/ProductReserve.jsx

```diff
@@ -36,7 +127,7 @@ const ProductReserve = ({ courtData }) => {
               id="noMatching"
               name="isMatch"
               defaultChecked
-              onChange={handleInput}
+              onChange={e => handleInput(e)}
```

Comment:
```suggestion
              onChange={handleInput}
```
이벤트 인자는 설정해 주지 않아도 자동으로 인지 하기 때문에
이렇게 작성하셔도 됩니다!